packages/vscode-extension/package.nls.json

```diff
@@ -474,5 +474,29 @@
+    "teamstoolkit.chatParticipants.officeAddIn.create.failToCreate": "Unable to create the project.",
+    "teamstoolkit.chatParticipants.officeAddIn.nextStep.promptAnswer": "This command provides guidance on your next steps based on your workspace.\n\nE.g. If you're unsure what to do after creating a project, simply ask Copilot by using @office /nextstep.",
+    "teamstoolkit.chatParticipants.officeAddIn.generateCode.description": "Use this command to generate code for the Office Add-ins.",
+    "teamstoolkit.chatParticipants.officeAddIn.generateCode.noPromptAnswer": "Use this command to provide description and other details about the code snippets you want to try.\n\nE.g. @office /generatecode I want to insert a content control in a Word document.\n\n@office /generatecode I want to insert a chart for the selected cells in Excel.",
+    "teamstoolkit.chatParticipants.officeAddIn.harmfulInputResponse": "Sorry, I can't assist with that.",
+    "teamstoolkit.chatParticipants.officeAddIn.default.noConceptualAnswer": "This is a procedural question, @office can only answer questions regarding descriptions or concepts for now. You could try these commands or get more info from [Office Add-ins documentation](https://learn.microsoft.com/office/dev/add-ins/overview/office-add-ins).\n\n • /create: Use this command to build your Office Add-ins as per your description. \n\n • /generatecode: Use this command to generate code for the Office Add-ins. \n\n • /nextstep: Use this command to move to the next step at any stage of your Office Add-ins development.",
+    "teamstoolkit.chatParticipants.officeAddIn.default.noJSAnswer": "This is question is not relevant with Office JavaScript Add-ins, @office can only answer questions regarding Office JavaScript Add-ins. You could try these commands or get more info from [Office Add-ins documentation](https://learn.microsoft.com/office/dev/add-ins/overview/office-add-ins).\n\n • /create: Use this command to build your Office Add-ins as per your description. \n\n • /generatecode: Use this command to generate code for the Office Add-ins. \n\n • /nextstep: Use this command to move to the next step at any stage of your Office Add-ins development.",
+    "teamstoolkit.chatParticipants.officeAddIn.default.canNotAssist": "I can't assist you with this request.",
+    "teamstoolkit.chatParticipants.officeAddIn.issueDetector.fixingErrors": "Attempting to fix code errors... ",
+    "teamstoolkit.chatParticipants.officeAddIn.printer.outputTemplate.intro": "For your question:",
+    "teamstoolkit.chatParticipants.officeAddIn.printer.outputTemplate.codeIntro": "Here is a code snippet using Office JavaScript API and TypeScript to help you get started:\n",
+    "teamstoolkit.chatParticipants.officeAddIn.printer.outputTemplate.ending": "The code above powered by AI, so surprises and mistakes are possible. Make sure to verify any generated code or suggestions.",
```

Comment:
Proposed: The above code is powered by AI, so mistakes are possible. Make sure to verify the generated code or suggestions.

Reply:
All fixed. Thanks for the suggestion.